lib/YuiRestClient.pm

```diff
@@ -65,8 +71,14 @@ sub setup_libyui {
 }
 
 sub teardown_libyui {
-    assert_screen('startshell', timeout => 100);
-    type_string_slow "exit\n";
+    if (get_var('S390_ZKVM')) {
+        wait_serial('ATTENTION: Starting shell', 120) || die "start shell didn't show up";
+        save_svirt_pty;
```

Comment:
Do we need this line? pty should be already set when we finish the installation.

Reply:
I was wondering the same, I will delete it.